src/OrchardCore/OrchardCore.Users.Core/Indexes/UserIndex.cs

```diff
@@ -9,6 +10,9 @@ public class UserIndex : MapIndex
         public string NormalizedUserName { get; set; }
         public string NormalizedEmail { get; set; }
         public bool IsEnabled { get; set; }
+        public bool LockoutEnabled { get; set; }
+        public DateTimeOffset? LockoutEnd { get; set; }
```

Comment:
I'm not sure, but I think we can't use DateTimeOffset with SQLite. Unless we find a fix for YesSQL. Normally a DateTime should be fine enough else we will need to save this as a string timestamp.